.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,128 @@
+name: continuous-integration
+
+# TODO: build master branch after PRs are merged.
+on:
+  schedule:
+    - cron: '0 2 * * *'
+    
+jobs:
+  windows:
+    name: Windows
+
+    runs-on: windows-2016
+
+    steps:
+     - uses: actions/checkout@v1
```

Comment:
Indentation is inconsistent throughout the file. (line 6 is indented 2 spaces, line 90 is indented 0 spaces from the previous line).

Reply:
Last edits I made were in web interface of github so using spaces only. I just followed the examples on github for indentation, if incorrect the script will not run.